lib/presentation/widget/upload_file/upload_file_widget.dart

```diff
@@ -79,7 +79,9 @@ class _UploadFileWidgetState extends State<UploadFileWidget> {
     uploadFileViewModel.setUploadFilesArgument(arguments.uploadFiles);
     uploadFileViewModel.setShareTypeArgument(arguments.shareType);
     uploadFileViewModel.setDocumentsArgument(arguments.documents);
-    uploadFileViewModel.setWorkGroupDocumentUploadInfoArgument(arguments.workGroupDocumentUploadInfo);
+    if (arguments.workGroupDocumentUploadInfo != null) {
+      uploadFileViewModel.setWorkGroupDocumentUploadInfoArgument(arguments.workGroupDocumentUploadInfo);
+    }
```

Comment:
can you explain the root cause of this?

Reply:
Oh the `build(BuildContext context)` method is called again, and set the argument to `null` while we already had data inside `workGroupDocumentUploadInfoArgument `